index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Md. Porag Sarkar: Portfolio</title>
+</head>
+
+<body>
+    <div class="container-parent">
+
+        <header class="top-navigation">
+            <div class="container-logo">
+                <p class="logo">My Logo</p>
+            </div>
+            <img class="menu-icon" src="./assets/mobile/Icon - Menu.svg" alt="Menu Button">
+
+        </header>
+
+        <main class="main">
+            <section class="headline">
+                <div class="text-primary">
+                    <h2 class="primary-one">I’m Porag. Glad to see you!</h2>
```

Comment:
- [ ] Since this is the main title of your portfolio, It will be nice to use an `h1` tag. That will allow google (and other search engines) to know about a specific topic, in this case, you. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.